sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -173,36 +172,36 @@ public String getChatThreadId() {
     /**
      * Adds participants to a thread. If participants already exist, no change occurs.
      *
-     * @param options Options for adding participants.
+     * @param participants Collection of participants to add.
      *
      * @return the {@link CompletableFuture} that signals the operation result.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public CompletableFuture<Void> addParticipants(AddChatParticipantsOptions options) {
-        if (options == null) {
-            return CompletableFuture.failedFuture(new NullPointerException("'options' cannot be null."));
+    public CompletableFuture<AddChatParticipantsResult> addParticipants(Iterable<ChatParticipant> participants) {
```

Comment:
What's the reason we went with `Iterable<ChatParticipant>` instead of something like  `List<ChatParticipant>`?

Reply:
This is from JAVA SDK. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadAsyncClient.java#L156

Reply:
It seems taking `Iterable` was added as part of a[ java-pr](https://github.com/Azure/azure-sdk-for-java/pull/19776) with title "Incoming Changes from Azure SDK review", could be that its a feedback from arch to take Iterable.